include/TMS/dev/TCA9545A.h

```diff
@@ -0,0 +1,62 @@
+#ifndef TMS_TCA9545A_H
+#define TMS_TCA9545A_H
+
+#include <cstddef>
+#include <EVT/io/I2C.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace TCA9545A {
+
+constexpr uint8_t TCA9545A_I2C_ADDR = 0x73; // Address can change based on pins
```

Comment:
Why would we pass the i2c address in as a part of the constructor?

Reply:
Because the TCA9545A can be configured with different slave addresses, so the class should be able to handle the possibility of having different slave addresses

Reply:
Makes sense